code/datums/cookingrecipes.dm

```diff
@@ -172,6 +172,12 @@ ABSTRACT_TYPE(/datum/cookingrecipe)
 	cookbonus = 6 // still mostly raw, since we don't kill it
 	output = /obj/item/reagent_containers/food/snacks/burger/aburgination
 
+/datum/cookingrecipe/itcamefromtheoven
+	item1 = /obj/item/material_piece/wad/blob
```

Comment:
should have dough in the recipe, to match other burgs

Reply:
the sprite is entirely blob colors, so it'd be a bit weird having dough in the recipe but no dough in the sprite

Reply:
the recipe also requires two chunks of blob because one is meant to be the "dough", and the other "meat"